fedimint-testing/src/ln/mod.rs

```diff
@@ -38,6 +41,7 @@ pub trait LightningTest: ILnRpcClient {
         // `FakeLightningTest` will fail to pay any invoice with
         // `INVALID_INVOICE_DESCRIPTION` in the description of the invoice.
         Ok(InvoiceBuilder::new(Currency::Regtest)
+            .payee_pub_key(kp.public_key())
```

Comment:
This will make it a little more obvious that invalid node pubkey is used. I know that's already the case, but this makes it a little more clear.